public/app/brothers/brothers.html

```diff
@@ -0,0 +1,15 @@
+<h1>All Brothers</h1>
+<div>
+<table>
+  <th>
+      <td>Brother Name</td>
+      <td>Active/Inactive</td>
+      <td>Pin</td>
+  </th>
+  <tr ng-repeat='brother in brothers'>
```

Comment:
we might want to use a directive for this, if it's going to look like the mockup I'll make tomorrow